C#/tests/syntax_test_Operators.cs

```diff
@@ -9,8 +9,12 @@
 Customer? first = customers?[0];
 /// <- support.type
 ///     ^ storage.type.nullable
-///                        ^^^^ meta.brackets
-///                        ^^ punctuation.section.brackets.begin
+///       ^^^^^ variable.other
+///             ^ keyword.operator.assignment.variable
+///               ^^^^^^^^^ variable.other
+///                        ^ keyword.operator.null-coalescing
```

Comment:
Should there be a `- meta.brackets` here, or does that make things more complicated?

Reply:
good idea, added :slightly_smiling_face: